consensus/poet/core/sawtooth_poet/poet_consensus/poet_block_verifier.py

```diff
@@ -123,18 +123,27 @@ def verify_block(self, block_wrapper):
                     utils.deserialize_wait_certificate(
                         block=block_wrapper,
                         poet_enclave_module=poet_enclave_module)
-                wait_certificate.check_valid(
-                    poet_enclave_module=poet_enclave_module,
-                    certificates=certificates,
-                    poet_public_key=validator_info.signup_info.poet_public_key)
+                if wait_certificate is None:
+                    raise \
+                        ValueError(
+                            'Being asked to verify a block that was not '
+                            'created by PoET consensus module')
+
+                try:
+                    poet_public_key = \
+                        validator_info.signup_info.poet_public_key
+                    wait_certificate.check_valid(
+                        poet_enclave_module=poet_enclave_module,
+                        certificates=certificates,
+                        poet_public_key=poet_public_key)
+                except ValueError as ve:
+                    LOGGER.error('Wait certificate is not valid: %s', str(ve))
```

Comment:
Technically, it is just a placeholder until the PoET consensus module places nice with the `poet genesis` comment (i.e., loading the sealed signup data that was generated so that it can seed the enclave with the keys).  That is in the PoET consensus state stuff I am working on now.